openapi/components/responses/evidence-list.yaml

```diff
@@ -2,240 +2,11 @@ description: A list of Learning Moments
-                      example: true
-                    award:
-                      nullable: true
-                      example: null
-                    recipient:
-                      $ref: ../schemas/userShort.yaml
-                    awardedBy:
-                      $ref: ../schemas/userShort.yaml
-                    awardedAt:
-                      type: string
-                      format: date-time
-                      description: |
-                        The date as a RFC3339 string.
-                      example: 2022-08-27T12:49:11+10:00
-                    expiresAt:
-                      nullable: true
-                      type: string
-                      format: date-time
-                      description: |
-                        The date as a RFC3339 string.
-                      example: null
-                    viewedOn:
-                      nullable: true
-                      type: string
-                      format: date-time
-                      description: |
-                        The date as a RFC3339 string.
-                      example: null
-                    _links:
-                      type: object
-                      properties:
-                        achievementWithdraw:
-                          type: string
-                          example: /learning/badge/award/10/achievement/withdraw
-                        goalWithdraw:
-                          type: string
-                          example: /learning/badge/award/10/goal/withdraw
-                        goalAward:
-                          type: string
-                          example: /learning/badge/award/10/achievement
-            date:
-                type: string
-                format: date-time
-                description: |
-                  The date as a RFC3339 string.
-                example: 2022-08-30T10:09:09+10:00
-        metadata:
-          type: object
-          properties:
-            count:
-              type: integer
-              example: 18
-            cursor:
-              type: object
-              properties:
-                current:
-                  type: string
-                  nullable: true
-                next:
-                  type: object
-                  description: |
-                    base64 encoded object
-                  properties:
-                    id:
-                      $ref: ../schemas/id.yaml
-                    discriminator:
-                      type: string
-                      enum:
-                        - evidence
-                        - badgeAward
-                      example: badgeAward
-                    date:
-                      type: string
-                      format: date-time
-                      description: |
-                        The date as a RFC3339 string.
-                      example: 2022-08-30T10:09:09+10:00
\ No newline at end of file
+      discriminator:
```

Comment:
The discriminator being applied here implies that the list consists of _only_ learning moments, or _only_ badge awards: it needs to contain both at once.
![image](https://user-images.githubusercontent.com/407053/190282013-408b6302-eb92-475b-a48b-75209fb4abb9.png)

Reply:
@JustinAClarke how'd you go with addressing this issue?

Reply:
[898db75](https://github.com/alaress/schoolbox-api-docs/pull/50/commits/898db752f2dba200303e1cde7842e5cf251460c2)

discriminator actually does it job

Reply:
To me this still implies the evidence may only be an array of learning moments, or an array of badge awards?

Reply:
Sorry @JustinAClarke I don't think what I wanted to say really came across.

The "one of" currently displays like this:
![image](https://user-images.githubusercontent.com/407053/191426183-00b44ed6-caa7-4ee7-a144-93807782df11.png)

It should be at the "object" level within "data", sort of like this (forgive the copypasted image):
![image](https://user-images.githubusercontent.com/407053/191426533-3757e2bd-4f5e-4351-a5d2-e42eb55a97f7.png)

Reply:
aha cool, yep, that makes sense...
i think I originally tried to do that, but it wasn't working, ill give it another go and see if I can fight my way through it

Reply:
Managed to get it to this stage, and this is working
![image](https://user-images.githubusercontent.com/4447143/192434917-ebea16e9-ad1b-4432-b927-6a7071025dab.png)
But the `discriminator` field is not a select box, and the main dropdown just lists 'Example #' which is not helpful...
![image](https://user-images.githubusercontent.com/4447143/192435063-4cd62e7d-7061-4852-9cf5-b105b3c7eaa8.png)

The schema tab does list the actual fields correctly.